--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@
 Priority: extra
 Maintainer: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
 Uploaders: Arne Bernin <arne@alamut.de>, Yaroslav Halchenko <debian@onerussian.com>, Mark Renouf <mark.renouf@gmail.com>
-Build-Depends: debhelper (>= 7.0.50), dpkg-dev (>= 1.16.1~), python-support,
+Build-Depends: debhelper (>= 7.0.50), python-support,
  cmake,  pkg-config, libusb-1.0-0-dev, freeglut3-dev | libglut3-dev,
  libxmu-dev, libxi-dev, python-dev, cython, python-numpy, doxygen
 Standards-Version: 3.9.2
--- a/debian/rules
+++ b/debian/rules
@@ -1,9 +1,9 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-DEB_LDFLAGS_MAINT_APPEND=-Wl,-Bsymbolic-functions -Wl,--as-needed
-include /usr/share/dpkg/buildflags.mk
-export CFLAGS CPPFLAGS LDFLAGS
+LFLAGS?=-Wl,-Bsymbolic-functions
+LDFLAGS+=-Wl,--as-needed
+export LDFLAGS
 
 %:
 	dh $@
@@ -11,7 +11,8 @@
 override_dh_auto_configure:
 	dh_auto_configure -- -DBUILD_PYTHON:Bool=True
 
-override_dh_auto_build-indep:
+override_dh_auto_build:
+	dh_auto_build
 ifeq (,$(filter nodoc,$(DEB_BUILD_OPTIONS)))
 	doxygen debian/doxygen.conf
 else
@@ -29,5 +30,6 @@
 override_dh_installdocs:
 	dh_installdocs --all CONTRIB
 
-override_dh_clean-indep:
+override_dh_clean:
+	dh_clean
 	rm -rf debian/html
